Description: Fix lubdir location in pkg-config and cmake files.
Author: Alexander Gerasiov <gq@debian.org>
Bug-Debian: https://bugs.debian.org/857729
Last-Update: 2017-03-27

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1358,7 +1358,7 @@
 
 set( prefix "${CMAKE_INSTALL_PREFIX}" )
 set( exec_prefix "\${prefix}" )
-set( libdir "\${exec_prefix}/${CMAKE_INSTALL_LIBDIR}" )
+set( libdir "${CMAKE_INSTALL_LIBDIR}" )
 set( includedir  "\${prefix}/include" )
 set( plugindir "\${libdir}/wireshark/plugins/${CPACK_PACKAGE_VERSION}" )
 
--- a/cmake/modules/WiresharkConfig.cmake.in
+++ b/cmake/modules/WiresharkConfig.cmake.in
@@ -7,7 +7,7 @@
 set(Wireshark_INSTALL_DIR        "@CMAKE_INSTALL_PREFIX@")
 set(Wireshark_PLUGIN_INSTALL_DIR "@PLUGIN_INSTALL_DIR@")
 
-set(Wireshark_LIB_DIR           "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@")
+set(Wireshark_LIB_DIR           "@CMAKE_INSTALL_LIBDIR@")
 set(Wireshark_INCLUDE_DIR       "@CMAKE_INSTALL_PREFIX@/include/wireshark")
 set(Wireshark_CMAKE_MODULES_DIR "@CMAKE_INSTALL_MODULES_DIR@")
 
